src/cider/nrepl/middleware/info.clj

```diff
@@ -57,9 +59,44 @@
           blacklist
           util/transform-value))))
 
+(defn extract-class-from-compliment [ns-str context]
```

Comment:
I docstring will be nice here, especially given this is a public function. (I know it's clear what it does, but still)